cypress/e2e/draw/draw-feat-style.cy.ts

```diff
@@ -0,0 +1,335 @@
+      cy.get('*[data-cy="featStyleSymbol_2"]').click()
+      cy.get('*[data-cy="featStyleSize"]').type('{selectAll}50')
+      cy.get('*[data-cy="featStyleAngle"]')
+        .find('input')
+        .eq(1)
+        .type('{selectAll}30', { force: true })
+    })
+  })
+
+  describe('When checking featureStyle in OL DrawnFeature object', () => {
+    it('feature style shall be applied', () => {
+      // get featureLayer
+      cy.window()
+        .its('olMap')
+        .then(function (olMap) {
+          const featureLayers = olMap
+            .getLayers()
+            .getArray()
+            .filter((l: any) => l.get('cyLayerType') == 'featureLayer')
+          const features = featureLayers
+            .map((l: any) => l.getSource().getFeatures())
+            .flat()
+          cy.wrap(features.length).should('equal', 1)
+
+          // const ff = features.find((f: any) => f.get('name') == 'Point 1')
+          const ff = features[0]
+          cy.wrap(ff.featureType).should('equal', 'drawnPoint')
+          // check color green
+          cy.wrap(ff.featureStyle.color).should('equal', '#00ff00')
+          // check shape: cross
+          cy.wrap(ff.featureStyle.shape).should('equal', 'cross')
+          // check size = 50
+          cy.wrap(ff.featureStyle.size).should('equal', 50)
+          // check angle = 30°
+          //cy.wrap(Math.abs(30 - ff.featureStyle.angle / Math.PI * 180)).should('be.lessThan', 0.0001)
+        })
+
+      // test cancel button
+      cy.get('button[data-cy="featureEditCancel"]').click()
+
+      cy.window()
+        .its('olMap')
+        .then(function (olMap) {
+          const featureLayers = olMap
+            .getLayers()
+            .getArray()
+            .filter((l: any) => l.get('cyLayerType') == 'featureLayer')
+          const features = featureLayers
+            .map((l: any) => l.getSource().getFeatures())
+            .flat()
+          cy.wrap(features.length).should('equal', 1)
+
+          const ff = features[0]
+          // check size = 10
+          cy.wrap(ff.featureStyle.size).should('equal', 10)
+        })
+    })
+  })
+})
+
+describe('Test style edition of Label feature', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    describe('Draw "Point"', () => {
+      cy.get('button[data-cy="drawButton"]').click()
+      cy.get('button[data-cy="drawLabelButton"]').click()
+      cy.get('div.ol-viewport').click(250, 150, { force: true })
+    })
+    describe('Edit feature style', () => {
+      cy.get('*[data-cy="featItemActionStyle"]').click()
+      cy.get('*[data-cy="featStyleColor"]')
+        .invoke('val', '#009f09')
+        .trigger('input')
+      cy.get('*[data-cy="featStyleSize"]').type('{selectAll}8') // max size 10
+      cy.get('*[data-cy="featStyleAngle"]').type('{selectAll}60')
+      cy.get('button[data-cy="featureEditValidate"]').click()
+    })
+  })
+
+  describe('When checking featureStyle in OL DrawnFeature object', () => {
```

Comment:
Actually e2e is meant to mimic user behavior in the browser, so put this statement "When checking featureStyle in OL DrawnFeature object" in a describe for an e2e tests is awkward.

Reply:
That was the result of some discussion, in theory e2e would be only capable of checking rendered pixels,  but this seems a bit too picky. Both checking th URL and checking the OL features' attributes seem a good compromise for meaningful tests

Reply:
I was talking about the text description, not the test itself.